### DISCUSSION TITLE
状态码3

### DISCUSSION THREAD

**Discussion by xyjk0511** [mikumifa/biliTickerBuy#325] [提问专区]
![6c58d513dc27b3294bd1b77450f2f0c9](https://github.com/mikumifa/biliTickerBuy/assets/127614382/0a99ceab-d8da-4d1e-b26e-3138c433e22f)
这个提示是什么原因 需要把间隔设长一点吗 现在是300ms

**Comment by mikumifa** [comment]
这个不管他
